File: Python/evenFibo.py
def evenFiboSum(n):
    arr = [1, 2]
    sum = 2

    while True:
        nextNo = arr[-1] + arr[-2]

        if nextNo > n:
            break

        arr.append(nextNo)

        if nextNo % 2 == 0:
            sum = sum+ nextNo
    
    print(sum) 



evenFiboSum(34)

File: JS/evenFibo.js
//Tabulation
function fiboEvenSum(n) {
    let arr = [1, 2];
    let evenSum = 2;

    while(true) {
        let nextNo = arr[arr.length-1] + arr[arr.length-2]

        if (nextNo > n) {
            break
        }

        arr.push(nextNo)

        if(nextNo%2 == 0) {
            evenSum += nextNo
        }
    }
    return evenSum


}

fiboEvenSum(34)